build/infrastructure/main/dbwj-integrationeventspersister.tf

```
@@ -33,24 +33,13 @@ resource "databricks_job" "integration_events_persister_streaming_job" {
       }
     }
 
-    library {
-      pypi {
-        package = "configargparse==1.2.3"
-      }
-    }
-
-    library {
-      pypi {
-        package = "azure-storage-blob==12.7.1"
-      }
-    }
-
     library {
       whl = "dbfs:/package/package-1.0-py3-none-any.whl"
     } 
 
-    spark_python_task {
-      python_file = "dbfs:/wholesale/integration_events_persister_streaming.py"
+    python_wheel_task {
+      package_name = "package"
+      entry_point = "start_stream"
```

Why not something more specific?
```suggestion
      entry_point = "integration_event_persister"
```

---

I would not call it that, we have that name everywhere already, and it will be difficult to know what a name refers to. We should however rename the package to integration_event_persister, but maybe in another PR.